lib/rubocop/cop/sorbet/signatures/enforce_signatures.rb

```diff
@@ -93,11 +93,11 @@ def check_node(node)
         end
 
         def param_type_placeholder
-          cop_config["ParameterTypePlaceholder"] || "T.untyped"
+          cop_config["ParameterTypePlaceholder"] || "T.untyped" # rubocop:todo InternalAffairs/UndefinedConfig
```

Comment:
Due to a bug, it doesn't seem possible to make this cop pass, so I'm disabling it inline.

#184 will disable it entirely until the bug is fixed, at which point we can address all the offenses.